src/stream/src/executor_v2/batch_query.rs

```diff
@@ -35,13 +35,14 @@ pub struct BatchQueryExecutor<S: StateStore> {
 
     info: ExecutorInfo,
 
-    #[allow(dead_code)]
     /// Indices of the columns on which key distribution depends.
     key_indices: Vec<usize>,
 
-    /// This is a workaround for parallelized chain.
-    /// TODO(zbw): Remove this when we support range scan.
-    parallel_info: BatchParallelInfo,
+    /// Which parallel unit this actor belongs to
+    parallel_unit_id: u32,
+
+    /// Consistent hash mapping for filtering data.
+    hash_mapping: Vec<u32>,
```

Comment:
These can be simplified into a list (or bitmap) of the **required** vnodes of this instance of BatchQueryExecutor

Reply:
Now batch query executor will build a bitmap from compressed hash mapping, with only required vnode set to one.